# HG changeset patch
# User Rik <rik@octave.org>
# Date 1356653395 28800
#      Thu Dec 27 16:09:55 2012 -0800
# Node ID 37371278f9fc9317a17c12959668b4cb6c078db5
# Parent  4294430ef736f35a87d8a775cb5f8090a1ded336
configure.ac: Remove unused EXTERN_CFLAGS, EXTERN_CXXFLAGS variables

* configure.ac: Remove unused EXTERN_CFLAGS, EXTERN_CXXFLAGS variables

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -13,23 +13,16 @@ dnl Copyright (C) 1993-2012 John W. Eato
 ### ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
 ### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 ### for more details.
 ### 
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <http://www.gnu.org/licenses/>.
 
-### Preserve CFLAGS and CXXFLAGS from the environment before doing
-### anything else because we don't know which macros might call
-### AC_PROG_CC or AC_PROG_CXX.
-
-EXTERN_CFLAGS="$CFLAGS"
-EXTERN_CXXFLAGS="$CXXFLAGS"
-
 AC_PREREQ([2.62])
 AC_INIT([GNU Octave], [3.7.0+], [http://octave.org/bugs.html], [octave])
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 OCTAVE_API_VERSION_NUMBER="48"
 OCTAVE_API_VERSION="api-v$OCTAVE_API_VERSION_NUMBER+"
 OCTAVE_RELEASE_DATE="2012-02-22"
